### PR TITLE
[JBPM-9738] Adding integrations test

### PR DESCRIPTION
**Pull request by fjtirado**
**JIRA**:

[link](https://issues.redhat.com/browse/JBPM-9738)

**referenced Pull Requests**: 
https://github.com/kiegroup/jbpm/pull/1992

### CODE REVIEW

**Comment by fjtirado** [comment]
related failures, debugging

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyProcessWithHeaders.bpmn2 [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        <bpmn2:dataOutput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ResultOutputX" drools:dtype="com.jboss.soap.service.acmedemo.Flight" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ResultOutputXItem" name="Result"/>
+        <bpmn2:inputSet id="_GgtfMotEEeiO7cOOapYI8w">
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ParameterInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_NamespaceInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ModeInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_UrlInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_InterfaceInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_OperationInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputX</bpmn2:dataInputRefs>
+        </bpmn2:inputSet>
```
Comment (elguardian) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyProcessWithHeaders.bpmn2:95]: I don't see the headers HEADER_prefix anywhere. where is it ?

Comment (fjtirado) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyProcessWithHeaders.bpmn2]: I messed up when saving the file, It is corrected now

**Review by gmunozfe — CHANGES_REQUESTED** [review]
Typos in header namespace param.
It could be great if outgoing headers are checked somehow (server responds based on the received header)

**Comment by fjtirado** [comment]
jenkins retest this

**Comment by fjtirado** [comment]
jenkins retest this

**Comment by mareknovotny** [comment]
jenkins retest this please

**Comment by mareknovotny** [comment]
@elguardian @gmunozfe @afalhambra all green so only remaining is your review/approval ;)

**Review by gmunozfe — APPROVED** [review]
Looks good to me, thanks for merging @fjtirado !

**Comment by elguardian** [comment]
marked as failure due to enviornment. test are ok.

**Pull request merged**